src/main/java/com/icapps/build/gradle/plugin/utils/ShellHelper.kt

```diff
@@ -1,20 +1,75 @@
 package com.icapps.build.gradle.plugin.utils
 
+import joptsimple.internal.Strings
+import org.gradle.internal.os.OperatingSystem
 import java.io.BufferedReader
+import java.io.File
 import java.io.InputStreamReader
 
 /**
  * @author Koen Van Looveren
  */
 object ShellHelper {
```

Comment:
This is a ShellHelper -> not specific git stuff
First item is always the command -> find path unless specified as relative (./)